sync_check_terraform/modules/sync_check/service/init.sh

```diff
@@ -10,3 +10,24 @@ dnf clean all
 gem install slack-ruby-client sys-filesystem
 
 nohup /bin/bash ./run_service.sh > run_service_log.txt &
+
+# Set-up the New Relic license key and custom configuration
+cat << EOF | sudo tee -a /etc/newrelic-infra.yml
+enable_process_metrics: true
+status_server_enabled: true
+status_server_port: 18003
+license_key: "$NR_LICENSE_KEY"
+custom_attributes:
+  nr_deployed_by: newrelic-cli
+display_name: "$NAME"
+EOF
+
+# Add the New Relic infrastructure monitoring agent repository to the system's list of yum repos.
```

Comment:
yum should not be used anymore in Fedora. If there's a compelling reason to use yum instead of dnf, it must be documented.